lib/swagger-module.ts

```diff
@@ -35,87 +40,87 @@ export class SwaggerModule {
     };
   }
 
-  public static setup(
-    path: string,
-    app: INestApplication,
-    document: OpenAPIObject,
-    options?: SwaggerCustomOptions
-  ) {
+  private static serveStatic(finalPath: string, app: INestApplication) {
```

Comment:
```suggestion
  private static serveStatic(finalPath: string, app: INestApplication | NestExpressApplication | NestFastifyApplication) {
```
I think you can use these types for the `app` param to avoid the `as NestFastifyApplication` below

Reply:
your suggestion doesn't work

I've came up with a bit cleaner solution than before